psi4/driver/schema_wrapper.py

```diff
@@ -633,6 +635,14 @@ def run_json_qcschema(json_data, clean, json_serialization, keep_wfn=False):
     if keep_wfn:
         json_data["wavefunction"] = _convert_wavefunction(wfn)
 
+    files = {
+        "psi4.grad": Path(core.get_writer_file_prefix(wfn.molecule().name()) + ".grad"),
+        "psi4.hess": Path(core.get_writer_file_prefix(wfn.molecule().name()) + ".hess"),
+        # binary "psi4.180.npy": Path(core.get_writer_file_prefix(wfn.molecule().name()) + ".180.npy"),
+        "timer.dat": Path("timer.dat"),
```

Comment:
If "it can't handle timer.dat," why is this here? Just in case the timer is in the current file?

Reply:
good catch. I wrote the PR blurb from memory w/o realizing the timer.dat line was uncommented in code. I just checked again, and it does work fine locally, so ok to leave as-is, or I can comment it if ppl prefer.